mods/default/nodes.lua

```diff
@@ -447,20 +448,27 @@ minetest.register_node("default:dirt_with_grass_footsteps", {
 		{name = "default_dirt.png^default_grass_side.png",
 			tileable_vertical = false}},
 	groups = {crumbly = 3, soil = 1, not_in_creative_inventory = 1},
-	drop = 'default:dirt',
+	drop = "default:dirt",
 	sounds = default.node_sound_dirt_defaults({
 		footstep = {name = "default_grass_footstep", gain = 0.25},
 	}),
 })
 
-minetest.register_node("default:dirt_with_dry_grass", {
```

Comment:
Node must be kept to avoid unknown nodes.

Reply:
done